src/modules/getGameId.js

```diff
@@ -0,0 +1,16 @@
+// const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+/* const getGameId = () => {
+  fetch(baseUrl + 'games/', {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'Gena game',
+    }),
+    headers: {
+      'content-type': 'application/json; charset=UTF-8',
+    },
+  }).then((res) => res.json())
+  .then((json) => console.log(json));
+};
+
+export default getGameId; */
```

Comment:
- [ ]  kindly remove any commented code / unused files in your repo to make look clean & professional